lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -28,14 +28,18 @@ import { AccountInfo } from "../account/AccountInfo";
 import { buildClientInfoFromHomeAccountId, buildClientInfo } from "../account/ClientInfo";
 import { CcsCredentialType, CcsCredential } from "../account/CcsCredential";
 import { ClientConfigurationError } from "../error/ClientConfigurationError";
+import { RequestValidator } from "../request/RequestValidator";
 
 /**
  * Oauth2.0 Authorization Code client
  */
 export class AuthorizationCodeClient extends BaseClient {
+    // Flag to indicate if client is for hybrid spa auth code redemption
+    private hybridSpa: boolean;
 
-    constructor(configuration: ClientConfiguration) {
+    constructor(configuration: ClientConfiguration, hybridSpa: boolean = false) {
         super(configuration);
+        this.hybridSpa = hybridSpa;
```

Comment:
Should we make this a part of the `ClientConfiguration` object?

Reply:
Not sure, looked at that, but didn't seem like the right. Don't have strong opinions (that I know of).

Reply:
I think it would be appropriate under `SystemOptions`. I'd prefer keeping configuration options together if we're doing it that way otherwise I'd also be fine with making this it's own `Client` if more separation is needed.

Reply:
I like the idea of making it a `ClientConfig` parameter.